documentation/docs/pipelines/abapEnvironment/introduction.md

```diff
@@ -0,0 +1,92 @@
+# ABAP Environment Pipeline
+
+The goal of the ABAP Environment Pipeline is to enable Continuous Integration for the SAP Cloud Platform ABAP Environment, also known as Steampunk.
+In the current state, the pipeline enables you to pull your Software Components to specifc systems and perform ATC checks. The following steps are performed:
+
+* Create an instance of the SAP Cloud Platform ABAP Environment
```

Comment:
"Create an instance of the SAP Cloud Platform ABAP Environment" --> "Create an instance of the SAP Cloud Platform ABAP Environment Service"

Reply:
done